### PR TITLE
Including Jins profile

### PR DESCRIPTION
**Pull request by jinsjob**
___
NB - by making a pull request you confirm agreement to the [Contributors Agreement](https://github.com/openlawnz/openlawnz-web/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by strum3nt** [comment]
On hold waiting for Lina to format image

File: src/jsx/pages/About.jsx [hunk 1]
```diff
@@ -207,6 +208,17 @@ class About extends Component {
...
 									/>
 								</div>
+								<div className="card-item-small">
+									<div>
+										<img src={Jins} alt="Jins Mary Job" />
+										<strong>Jins Mary Job</strong>
+									</div>
+									<ReadMore
+										text="
+										Jins is software engineer, who is passionate about applying technology to solve complex problems and challenges. She loves coding and learning new technologies. She has Masters in computer applications and extensive experience in Microsoft technologies
```
Comment (dlouise64) [inline src/jsx/pages/About.jsx]: Missing `a` and remove `,`.
`Jins is software engineer, who` should be, `Jins is a software engineer who`
also here: `She has Masters` should be `She has Masters`

👍

Comment (jinsjob) [reply src/jsx/pages/About.jsx]: Thanks for the review comments. Corrected.

**Comment by caionnx** [comment]
@jinsjob @strum3nt @dlouise64 few additional commits to merge changes from #187 ready to merge now :+1: 

@jinsjob for the next task, please clone the project in your machine instead of forking the project, this improves how the PR is tested and how others can make changes. Here are some tips https://github.com/openlawnz/openlawnz-web/wiki/Workflow

**Pull request merged**